namer/namer.cc

```diff
@@ -2020,6 +2020,20 @@ class TreeSymbolizer {
             }
         }
 
+        // Prevent a crash by validating the arguments before we do any symbol lookups or manipulations
+        if (send->hasPosArgs() || send->hasKwArgs() || send->block() != nullptr) {
+            if (send->numPosArgs() > 1) {
+                if (auto e = ctx.beginError(send->loc, core::errors::Namer::InvalidTypeDefinition)) {
+                    e.setHeader("Too many args in type definition");
+                }
+                auto send = ast::MK::Send0Block(asgn.loc, ast::MK::T(asgn.loc), core::Names::typeAlias(),
+                                                asgn.loc.copyWithZeroLength(),
+                                                ast::MK::Block0(asgn.loc, ast::MK::Untyped(asgn.loc)));
+                return handleAssignment(
+                    ctx, ast::make_expression<ast::Assign>(asgn.loc, std::move(asgn.lhs), std::move(send)));
```

Comment:
For consistency, we usually like to use the helpers in ast/Helpers.h, so this would be ast::MK::Assign(…)

You can also get away with not using std:: on the calls to move if you want.